data/gegno/weapons.txt

```diff
@@ -0,0 +1,321 @@
+outfit "Irate Cannon"
+	category Guns
+	cost 475000
+	mass 2
+	thumbnail "outfit/irate cannon"
+	"gun ports" -1
+	"outfit space" -12
+	"weapon capacity" -12
+	licenses
+		"Gegno Civilian" 
+		"Vi Evocati"
+	weapon
+		sprite projectile/irate
+		sound irate
+		"fire effect" "irate fire"
+		"hit effect" "tiny explosion"
+		lifetime 15
+		reload 20
+		velocity 22
+		"hardpoint offset" 1.5
+		inaccuracy 2
+		"firing energy" 9
+		"firing heat" 155
+		"shield damage" 6
+		"hull damage" 16
+		"hit force" 40
+	description "Irate Cannons are a rudimentary cannon technology that fire explosive projectiles made out of condensed energy and metal, making them effective against armor. They fire decently fast, but one cannon alone is barely enough to output any sort of meaningful damage, which is why they are often found clustered together in numbers."
+
+outfit "Chloeric Cannon"
+	category Guns
+	cost 610000
+	mass 13
+	thumbnail "outfit/chloeric cannon"
+	"gun ports" -1
+	"outfit space" -37
+	"weapon capacity" -37
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+	weapon
+		sprite projectile/chloeric
+		sound chloeric
+		"fire effect" "chloeric fire"
+		"hit effect" "small explosion"
+		lifetime 55
+		reload 90
+		"missile strength" 18
+		velocity 11
+		"hardpoint offset" -11
+		inaccuracy 2
+		"firing energy" 35
+		"firing heat" 365
+		"shield damage" 95
+		"hull damage" 380
+		"hit force" 180
+	description "A caliber higher than the Irate Cannon, Chloeric cannons fire larger, slower rounds that do a noticeable chunk of damage to a ship's hull. However, due to the nature of the cannon (as well as its sister cannons), it is very ineffective against any sort of shielding."
+	description "	A flaw in the shell design, however, is that point defense systems are just able to make them out as valid targets on account of their slower speed, material make-up, and size nearing a small missile."
+
+
+outfit "Chloeric Turret"
+	category Turrets
+	cost 816000
+	mass 42

```

Comment:
The intention behind this is if they were to have the mass equal to their tonnage, they would make the ships *way* slower than I'd like them to be. Perhaps it could be an offset of the guns being outside the hull. 

Taking another look though, they are significantly lower. I think fixing them will be fine, especially because the Vi win in a straight up brawl typically more often than the Scin due to their teeth guns.